src/CMakeLists.txt

```diff
@@ -37,12 +37,9 @@ endif (WIN32)
 # generate list of common source files
 #-----------------------------------------------------------------------------
 set (NAPISRC napi.c napiu.c nxstack.c nxstack.h stptok.c  nxdataset.c 
+             napi_fortran_helper.c
```

Comment:
@FreddieAkeroyd I would not add `napi_fortran_helper.c` to the sources if it is not required. Is there a good reason why you put it there independently of whether or not Fortran bindings are enabled?